src/api/app/jobs/consistency_check_job.rb

```diff
@@ -1,61 +1,37 @@
 require 'api_error'
-require 'xmlhash'
 
 class ConsistencyCheckJob < ApplicationJob
   queue_as :consistency_check
 
   def perform
-    User.get_default_admin.run_as { _perform(nil) }
+    User.get_default_admin.run_as { _perform(false) }
   end
 
-  def check_one_project(project, fix)
-    unless Project.valid_name?(project.name)
-      @errors << "Invalid project name #{project.name}\n"
-      if fix
-        # just remove it, the backend won't accept it anyway
-        project.commit_opts = { no_backend_write: 1 }
-        project.destroy
-      end
-      return
-    end
-    @errors << package_existence_consistency_check(project, fix)
-    @errors << project_meta_check(project, fix)
+  def check_one_project(project, fix = false)
+    package_existence_consistency_check(project, fix)
+    project_meta_check(project, fix)
+    @errors.flatten.reject(&:empty?).join("\n")
   end
 
-  def _perform(fix)
-    @errors = project_existence_consistency_check(fix)
+  def _perform(fix = false)
```

Comment:
I don't like the name `_perform` . Any idea?

Reply:
What about `check_projects_in_batches`?

Reply:
IMO it cannot be `check_*` because it can be used to "fix" the errors too

Reply:
Then if this is doing two different things, maybe its better to split the behaviour, one checker and one fixer...